.osparc/docker-compose.overwrite.yml

```diff
@@ -4,3 +4,6 @@ services:
     build:
       dockerfile: docker/custom/Dockerfile
       target: production
+      args: 
+        - KWAVE_LOGIN
```

Comment:
how are you passing these?? They should go as environs on the service, no?

Reply:
Nope @pcrespov, I don't think this are environs of the service.
I use them during building to download the executable of this tool from a website that requires login. See here:
https://github.com/ITISFoundation/k-wave/blob/first_version/docker/custom/Dockerfile#L49